.travis.yml

```diff
@@ -33,6 +34,7 @@ matrix:
   allow_failures:
   - env: RUN_TEST=uitest-phantom START_SERVER=1 UPLOAD_CAPTURES=1 MAKE_LANGPACKS=1
   - env: RUN_TEST=sherlocked API=mock START_SERVER=1 MAKE_LANGPACKS=1
+  - env: RUN_TEST=uitest-webqa START_SERVER=1
```

Comment:
Will adding `API=dev` here make the server point to -dev on Travis? We may need to explicitly pass that envvar along but hopefully it just works.